README.md

```diff
@@ -90,26 +88,41 @@ Want to make your own plugin?
 Check out the documentation on [Writing Plugins](/docs/writing-plugins.md) for where to start.
 
 
-## Ubuntu setup
+## Installation
+
+Clusterio runs on Node.js v12 and up, it's distributed via npm and comes with a guided install script.
+If you alread have a recent Node.js installed you can set it up in a new directory with.
+
+    npm init "@clusterio"
```

Comment:
Are the quotes required here? Worked fine when I left it out.

Reply:
It's required in PowerShell, which does something special with `@name`, so I've added them to all of the guides so there's no confusion.